execution_workflows/QAPA/modules/gtf_to_genepred.nf

```diff
@@ -0,0 +1,22 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process GTFTOGENEPRED {
+    publishDir "${params.outdir}/pipeline_info", mode: params.publish_dir_mode
+    container "docker.io/apaeval/gtftobed:1.0"
+
+    input:
+    path gtf
+    
+    output:
+    path "*.genePred", emit: genepred
+    
+    script:
+    genepred= 'reference.genePred'
+    """
+    /gtfToGenePred -genePredExt $gtf $genepred
+    """
+}
```

Comment:
Like the re-use of our one and only utils :smile:

Reply:
Yup, this made my life so much easier without making a docker container :)